README.markdown

```diff
@@ -43,11 +43,8 @@ gem "rack-timeout", require:"rack/timeout/base"
 ```ruby
 # config/initializers/rack_timeout.rb
 
-#  insert middleware wherever you want in the stack
-Rails.application.config.middleware.insert_before Rack::Runtime, Rack::Timeout
-
-# customize seconds before timeout
-Rack::Timeout.timeout = 5
+# insert middleware wherever you want in the stack, optionally pass initialization arguments
+Rails.application.config.middleware.insert_before Rack::Runtime, Rack::Timeout, service_timeout:5
```

Comment:
Sort of a nit, but it's pretty standard to space out the optional parameters in Ruby. e.g. `service_timeout: 5`